DSCResources/cChocoPackageInstall/cChocoPackageInstall.schema.mof

```diff
@@ -6,4 +6,5 @@ class cChocoPackageInstall : OMI_BaseResource
   [write] string Params;
   [write] string Version;
   [write] string Source;
+  [Write,ValueMap{"allowunofficial","prerelease","forcex86","allowdowngrade","sidebyside","ignoredependencies","ignorechecksums","allowemptychecksum","allowemptychecksumsecure"},Values{"allowunofficial","prerelease","forcex86","allowdowngrade","sidebyside","ignoredependencies","ignorechecksums","allowemptychecksum","allowemptychecksumsecure"}] String chocoSwitchParams[];

```

Comment:
And as we talked about in #52 - probably removing the value maps entirely and let Chocolatey do the validation of the arguments.